dotnetv3/SES/README.md

```diff
@@ -0,0 +1,61 @@
+# Amazon Simple Email Service code examples for the SDK for .NET
+
+## Overview
+The examples in this section show to use the AWS SDK for .NET with Amazon Simple Email Service (Amazon SES).
+
+Amazon SES is a reliable, scalable, and cost-effective email service. Digital marketers and application developers can use Amazon SES to send marketing, notification, and transactional emails.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+## Code examples
+
+### Single actions
+Code excerpts that show you how to call individual service functions.
+* [Create an email template](Actions/SESActionExamples.cs) (CreateTemplate)
+* [Delete an email template](Actions/SESActionExamples.cs) (DeleteTemplate)
+* [Get the status of an identity](Actions/SESActionExamples.cs) (GetIdentityVerificationAttributes)
```

Comment:
Missing "Delete an identity".
Also missing "Get sending limits" if you decide to add it.

Reply:
Fixed.